src/plugins/guided_onboarding/public/services/api.service.ts

```diff
@@ -360,7 +361,8 @@ export class ApiService implements GuidedOnboardingApi {
    */
   public async completeGuideStep(
     guideId: GuideId,
-    stepId: GuideStepIds
+    stepId: GuideStepIds,
+    params?: GuideParams
```

Comment:
[nit] Probably outside the scope of this PR, but this might be easier to reason with if we passed in an single object with the arguments.

Reply:
👍 makes sense